pkg/TIPA.py

```
@@ -164,587 +191,489 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
 
-                        except:
-                            if tryCounter == 1:
-                                URL = URL_org
+    def mse(self, imageA: np.ndarray, imageB: np.ndarray) -> float:
+        '''
+        The 'Mean Squared Error' between the two images is the
+        sum of the squared difference between the two images;
+        NOTE: the two images must have the same dimension
+        '''
+        if imageA.shape != imageB.shape:
+            raise ValueError("Input images must have the same dimensions")
```

Ruff (0.8.0)

Avoid specifying long messages outside the exception class

(TRY003)